config/routes.rb

```diff
@@ -0,0 +1,22 @@
+
+Rails.application.routes.draw do
+
+  get "top_media/index"
```

Comment:
You don't need this route if you're using this controller/method for the homepage.